packages/wonder-blocks-announcer/src/init-announcer.ts

```diff
@@ -0,0 +1,18 @@
+import Announcer from "./announcer";
+
+type InitAnnouncerProps = {
+    debounceThreshold?: number;
+};
+
+/**
+ * Utility to inject Announcer on page load.
+ * It can be called from useEffect or elsewhere to improve ARIA Live Region performance on the first announcement.
+ * @returns {Announcer} The Announcer instance created.
+ */
+export function initAnnouncer(props?: InitAnnouncerProps): Announcer {
+    const announcer = Announcer.getInstance();
+    if (props?.debounceThreshold !== undefined) {
+        announcer.updateWaitThreshold(props?.debounceThreshold);
```

Comment:
Since there can only be one instance of the announcer, could it cause issues if different consumers called `initAnnouncer` with different `debounceThreshold` values? The last thing to call `initAnnouncer` would override the previous value

Reply:
Yes, that's a good point. It could change on every call.. but I think that is what you would want. Maybe we could play with it more in webapp and see if that becomes an issue?